include/solvers/vof.h

```diff
@@ -98,13 +100,19 @@ class VOF : public AuxiliaryPhysics<dim, TrilinosWrappers::MPI::Vector>
             SolutionTransfer<dim, TrilinosWrappers::MPI::Vector>(
               this->dof_handler));
       }
+
+    // Check the value of interface sharpness
+    if (simulation_parameters.interface_sharpening.interface_sharpness < 1.0)
+      this->pcout
+        << "Warning: interface sharpness values smaller than 1 smooth the interface instead of sharpening it."
+        << std::endl;
```

Comment:
```suggestion
        << "Warning: interface sharpness values smaller than 1 smooth the interface instead of sharpening it."
        << std::endl
        << "The interface sharpness value should be set between 1 and 2" << std::endl;
```